src/moscot/problems/time/_mixins.py

```diff
@@ -740,23 +740,23 @@ def _interpolate_gex_randomly(
 
     @staticmethod
     def _get_interp_param(
-        start: K, intermediate: K, end: K, interpolation_parameter: Optional[float] = None
+        source: K, intermediate: K, target: K, interpolation_parameter: Optional[float] = None
     ) -> Numeric_t:
         if TYPE_CHECKING:
-            assert isinstance(start, float)
+            assert isinstance(source, float)
             assert isinstance(intermediate, float)
-            assert isinstance(end, float)
+            assert isinstance(target, float)
         if interpolation_parameter is not None:
             if 0 < interpolation_parameter < 1:
                 return interpolation_parameter
             raise ValueError(
                 f"Expected interpolation parameter to be in interval `(0, 1)`, found `{interpolation_parameter}`."
             )
 
-        if start < intermediate < end:
-            return (intermediate - start) / (end - start)
+        if source < intermediate < target:
+            return (intermediate - source) / (target - source)
         raise ValueError(
-            f"Expected intermediate time point to be in interval `({start}, {end})`, found `{intermediate}`."
+            f"Expected intermediate time point to be in interval `({source}, {target})`, found `{intermediate}`."
```

Comment:
Can be `{source, target}`

Reply:
It will be still formatted the same, though slightly more efficient, but I agree, it's more confusing, let's keep as it is.